transformer_engine/jax/cpp_extensions/misc.py

```diff
@@ -167,3 +170,8 @@ def is_ffi_enabled():
     is_enabled = int(os.getenv("NVTE_JAX_WITH_FFI", "1"))
     assert is_enabled in (0, 1), "Invalid NVTE_JAX_WITH_FFI value"
     return is_supported and is_enabled
+
+
+def is_bf16_compatible() -> None:
```

Comment:
I have not yet found where this helper is used.

Reply:
Removed - stale code from earlier